.github/workflows/ci.yml

```
@@ -0,0 +1,11 @@
+on: push
```

Maybe add `workflow_dispatch:` here so that it can be started manually?
```suggestion
on:
  push:
  workflow_dispatch:
```

---

Probably doesn't hurt :)